examples/faster_rcnn/README.md

```diff
@@ -2,11 +2,25 @@
 
 ### Performance
 
-| Training Setting | Evaluation | Reference Implementation | ChainerCV |
+| Training Setting | Evaluation | Reference | ChainerCV |
 |:-:|:-:|:-:|:-:|
 | VOC 2007 trainval | VOC 2007 test|  69.9 mAP [1] | 70.5 mAP |
 
 
+### Speed
+
+Speed of the Caffe implementation and ChainerCV implementation are almost same in the test mode.
```

Comment:
The subject of this sentence is `speed`, which is a singular form. And the verb of it is `are`, which is a plural form. Is it OK? I mean `Speed ... is` or `Speeds ... are` are better.